tests/enums/test_query_types.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from enums.query.query_types import QueryTypes
+from exceptions.parse_query_error import ParseQueryError
+
+
+@pytest.mark.parametrize("query_type", ["Flavor_Query", "FlAvOr_Query", "flavor_query"])
+def test_flavor_query_serialization(query_type):
+    """
+    Tests that variants of FLAVOR_QUERY can be serialized
+    """
+    assert QueryTypes.from_string(query_type) is QueryTypes.FLAVOR_QUERY
```

Comment:
We only need a single case, since they all use the same code path and will throw:

```
@pytest.mark.parameterize("query_type", ["Flavor_Query", "SERVER_QUERY"]
```

Would cover all of our cases, since we shouldn't need to validate `enum` works (as it's not our code)

Reply:
I'm leaving this as is - its the same way we tested other enums with `from_string` method. 
These tests can be adapted in the future if we decide to add aliases for query types same as aliases for properties/presets